src/main/java/com/pandora/backend/controller/AccountsController.java

```diff
@@ -25,13 +27,17 @@ public Account get(@PathVariable Integer id) {
     }
 
     @PostMapping
-    public Account post(@RequestBody Account account) {
```

Comment:
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk to chapado